pkg/driver/mounter.go

```diff
@@ -16,8 +19,19 @@ type Config struct {
 	Filer string
 }
 
+type Mount interface {
```

Comment:
Unmounter?

Reply:
I hope that 'Mount' interface will have some more functions in near future (i.e. 'FixMount'), but for now it may be 'Unmounter'. I will make the change.